.gitignore

```diff
@@ -0,0 +1,36 @@
+.gradle
+build/
+!gradle/wrapper/gradle-wrapper.jar
+!**/src/main/**/build/
+!**/src/test/**/build/
+
+### STS ###
+.apt_generated
+.classpath
+.factorypath
+.project
+.settings
+.springBeans
+.sts4-cache
+bin/
+!**/src/main/**/bin/
+!**/src/test/**/bin/
+
+### IntelliJ IDEA ###
+.idea
+*.iws
+*.iml
+*.ipr
+out/
+!**/src/main/**/out/
+!**/src/test/**/out/
+
+### NetBeans ###
```

Comment:
Why do you need `NetBeans`... Who is using NetBeans in 2021?

Reply:
It was generated by [Spring Initializr](https://start.spring.io/), but we can simplify it by leaving the Gradle and IDEA sections

Reply:
4b51eb6